### ISSUE TITLE
Making the Diagnostics Logs more Cloud-friendly

### ISSUE THREAD

**Issue by ncherkas** [hazelcast/hazelcast#16844]
In the Cloud-native K8S environment, the logging architecture almost always assumes that all needed logs are sent to the stdout. It works as a unified source of logs where different tools read them, re-organize if needed, and route to the destinations like Analytics Dashboards etc.

Hazelcast Diagnostics are very useful when troubleshooting the performance and stability issues but currently, it supports only the rotated file storage. This causes invention of different tricks and workarounds to somehow send these files to the stdout. 

It would be really helpful if we have a configuration setting that would sent all the Diagnostics logs to the stdout, instead of storing on the file system. 

One additional request to it is to add one more field to the output for easier filtering the Diagnostics from other logs e.g:

```
01-04-2020 09:00:03 1585731603216 HazelcastDiagnostics Metric[proxy.proxyCount=0]
01-04-2020 09:00:03 1585731603216 HazelcastDiagnostics Metric[runtime.availableProcessors=1]
01-04-2020 09:00:03 1585731603216 HazelcastDiagnostics Metric[runtime.freeMemory=4778378568]
01-04-2020 09:00:03 1585731603216 HazelcastDiagnostics Metric[runtime.maxMemory=6227755008]
01-04-2020 09:00:03 1585731603216 HazelcastDiagnostics Metric[runtime.totalMemory=6227755008]
01-04-2020 09:00:03 1585731603216 HazelcastDiagnostics Metric[runtime.uptime=442120]
01-04-2020 09:00:03 1585731603216 HazelcastDiagnostics Metric[runtime.usedMemory=1449376440]
...
```

**Comment by buraksezer** [comment]
Hi team, I would like to take this issue, if it's possible.

**Comment by Holmistr** [comment]
Hi @buraksezer , of course! We're glad to have you! :) Please don't hesitate to ask any questions. We're more than happy to help you. For faster feedback loop, please join our Gitter channel https://gitter.im/hazelcast/hazelcast .

**Comment by buraksezer** [comment]
Thank you! I work on it.

**Comment by mmedenjak** [comment]
Closing as fixed in 4.1
